app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ReviewsController < ApplicationController
+  before_action :require_login
+
+  def create
+    @video = Video.find_by slug: params[:video_id]
+    @review = @video.reviews.new review_params.merge!(user: current_user)
+    if @review.save
+      redirect_to video_path(@video), notice: 'Review saved.'
+    else
+      flash.now[:error] = 'Something is wrong with your submission.'
```

Comment:
This app uses bootstrap 3 which uses `success` and `danger` instead of `notice` and `error`.